Katello/katello
# [CP] Katello 4.3.1 cherrypicks and changelog

#### What are the changes introduced in this pull request?
CPs for Katello 4.3.1

#### Considerations taken when implementing this change?
There were a lot of CPs!  I had to manually edit a little bit of React code due to some conflicts, but it was all very basic.  Most of it was related to the recent linting fixes on top of some other React code that isn't making it into Katello 4.3, so mostly deletions rather than re-writing code.

#### What are the testing steps for this pull request?
1) Check that the tests pass.
2) Check that I didn't somehow bump to the wrong version
3) Check that the CHANGELOG looks okay.
